service/reducers/entities/helpers.js

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
+import {Constants} from 'service/constants';
+
 function isPostListNull(pl) {
```

Comment:
I can't remember if I've asked this before, but why are all of these functions in helpers.js when they'll only be used in posts.js?